study-builder/tenants/testboston/tenant.yaml

```diff
@@ -44,6 +60,7 @@ clients:
       ##LOCALHOST_CALLBACKS##
     allowed_logout_urls:
       - '##BASE_URL##'
+      - '##BASE_URL##/error'
```

Comment:
When someone attempts to login with google-oauth2, they will fail login if they're not allowed, so we redirect them to an error page. Need to whitelist the URL otherwise Auth0 won't redirect to it after logout.

Reply:
Where is this error page being generated from? Would it be a better option to return a machine-readable error code in the response from Auth0 and let the front-end handle the error entirely on its end?

Reply:
That is actually what we have. Auth0 will return an error, and the apps decide what they want to do. The error page (even it's URL) is defined by the apps.